docs/userGuide/syntax/modals.mbdf

```diff
@@ -22,6 +22,15 @@ This is the same <trigger for="modal:loremipsum">trigger</trigger> as last one.
   Centered
 </modal>
 
+<trigger for="modal:ok-text">This is a trigger for a modal with a custom OK button</trigger>.
+
+<modal header="OK button visible!" id="modal:ok-text" ok-text="Custom OK">
+  Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore
+  magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+  consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+  Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+</modal>
+
 ```
 </span>
 <span id="output">
```

Comment:
should `ok-text` be `ok-title` instead? ( the props table )

Reply:
Should be fine as `ok-text`. since it's not really a title of anything.

Reply:
> Remove the redundant OK button from modals, only show it if the author specifies `ok-title.`

Ahh ok, thought the attribute from the user standpoint here was `ok-title`